src/cookbook.js

```diff
@@ -7,7 +7,8 @@ class Cookbook {
     return this.recipes.filter(recipe => {
       return recipe.ingredients.find(ingredient => {
         return (ingredient.name.includes(searchText)) ||
-        (recipe.name.includes(searchText))
+        (recipe.name.includes(searchText)) ||
+        (recipe.tags.includes(searchText))
```

Comment:
I agree about displaying tags. I see this frequently as a user of cooking websites. Maybe they would display on the recipe card rather than the home page?